test/security_test/createkey.py

```diff
@@ -26,14 +26,14 @@ def init_params(appid, apikey, timestamp_id, keyspec, origin, sign=None):
     payload["origin"] = origin
     params = OrderedDict()
     params["appid"] = appid
-    params["payload"] = urllib.parse.unquote(urllib.parse.urlencode(payload))
+    params["payload"] = urllib.parse.unquote_plus(urllib.parse.urlencode(payload))
```

Comment:
Why use unquote_plus? The difference is that unquote_plus also changes + to space.

Reply:
unquote_plus can fix the space sign error.

Reply:
is there an example to show the error we met?

Reply:
on secretData, if you put "test data 1", on unquote will be sign error.